src/log/broadcast_backend.cr

```diff
@@ -0,0 +1,31 @@
+# A backend that broadcast to others backends.
+# Each of the referenced backends may have a different severity level filter.
+#
+# When this backend level is set that level setting takes precedence
+# over the severity filter of each referenced backend.
+#
+# This backend is not to be used explictly. It is used by `Log::Builder` configuration
+# to allow a given source to emit to multiple backends.
+class Log::BroadcastBackend < Log::Backend
+  property level : Severity? = nil
```

Comment:
```suggestion
  property level : Severity?
```

Reply:
Why?

Reply:
Because it's redundant?

Reply:
That may be, but more importantly it's explicit about the default value.

Reply:
`Explicit > Implicit` argument, let it be.